abmodel/tests/utils/test_column_checkers.py

```diff
@@ -1,23 +1,73 @@
-from unittest import TestCase
+import pytest
 
 import numpy as np
 import pandas as pd
 
-from abmodel.utils.utilities import check_column_errors, check_column_existance
+from abmodel.utils.utilities import check_field_errors, check_field_existance
 
 
-class ColumnErrorsTestCase(TestCase):
+class TestCaseFieldExistenceAndErrors():
```

Comment:
Remove unnecessary ().
In a class declaration, you should avoid empty parentheses ().